CONTRIBUTING.md

```diff
@@ -17,6 +17,20 @@ and what are the requirements for a Pull Request to be opened against Raiden.
 There are two ways you can contribute to the development. You can either open
 an Issue or if you have programming abilities open a Pull Request.
 
+## Setup development environment
+### Installation
+- Install [pyenv](https://github.com/pyenv/pyenv) and [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) and create a separate virtual environment for your raiden installation
+- Make sure that [tox](https://tox.readthedocs.io/en/latest/) is installed
+- Install [solidity](https://solidity.readthedocs.io/en/latest/installing-solidity.html) and ensure that ```solc --version``` prints out something
+
+### Dependencies
+- Run ```pip install -e .``` to install the dependencies
+
+### Tests
+- Run ```tox``` to run the tests.
+// TODO: Fix that after this some tests are still failing. 
```

Comment:
I'd rather not have an empty section with a TODO here. Tests may always fail locally, or be broken due to some mistake. That should not be reflected in the `README` file.

We have a strict rule of not accepting PRs if tests fail in the CI, so if tests fail locally it's probably something on your side.